connect-examples/v2/python_payment/cgi-bin/process_card.py

```diff
@@ -3,27 +3,41 @@
 from __future__ import print_function
 import uuid
 import cgi
+import ConfigParser
 
 import squareconnect
 from squareconnect.rest import ApiException
 from squareconnect.apis.transactions_api import TransactionsApi
 from squareconnect.apis.locations_api import LocationsApi
 
+# To read your secret credentials
+config = ConfigParser.ConfigParser()
+config.read('config.ini')
+
 # Create instance of FieldStorage
 form = cgi.FieldStorage()
 # Get data from fields
 nonce = form.getvalue('nonce')
 
 # The access token to use in all Connect API requests. Use your *sandbox* access
 # token if you're just testing things out.
-squareconnect.configuration.access_token = 'REPLACE_ME'
+if config.get("DEFAULT", "is_prod") == "true":
+    access_token = config.get("PRODUCTION", "access_token")
+else:
+    access_token = config.get("SANDBOX", "access_token")
+
+squareconnect.configuration.access_token = access_token
 
 # The ID of the business location to associate processed payments with.
 # See [Retrieve your business's locations]
 # (https://docs.connect.squareup.com/articles/getting-started/#retrievemerchantprofile)
```

Comment:
I would rewrite two if blocks something like below.
```
config_type = "PRODUCTION" if config.get("DEFAULT", "is_prod") == "true" else "SANDBOX"
access_token = config.get(config_type, "access_token")
location_id = config.get(config_type, "location_id")
```

Reply:
Updated to that style.